core/widget.php

```diff
@@ -182,8 +186,9 @@ public function save($required_details_form, $widget_options_form, $position_opt
 			// new
 			$params = array($this->state, $this->type_id, $this->title, $this->note, $options_json, $this->position_control, $this->global_position, implode(',',$this->page_list)) ;
 			$result = DB::exec("insert into widgets (state,type,title,note,options,position_control,global_position,page_list) values(?,?,?,?,?,?,?,?)", $params);
+			$new_widget_id = DB::get_last_insert_id();
 			if ($result) {
-				CMS::Instance()->queue_message('New widget saved','success',Config::uripath() . '/admin/widgets/show');	
+				CMS::Instance()->queue_message('Widget <a href="' . Config::uripath() . '/admin/widgets/edit/'.  $new_widget_id . '">' . $this->title .'</a> created','success', $redirect_url);	
```

Comment:
templating

Reply:
fixed in b772c9972b1b6528cd219293d837976ffad29f09